settings.go

```diff
@@ -109,8 +138,6 @@ func (s *Settings) Scope(db *gorm.DB, request *goyave.Request, dest interface{})
 			fields = modelIdentity.addForeignKeys(fields)
 		}
 		paginator.DB = paginator.DB.Scopes(selectScope(modelIdentity, modelIdentity.cleanColumns(fields, s.FieldsBlacklist)))
-	} else {
-		paginator.DB = paginator.DB.Scopes(selectScope(modelIdentity, s.getSelectableFields(modelIdentity.Columns)))
```

Comment:
Why is it removed? This prevents blacklist from working properly when no "fields" query param is provided.